src/lib/components/Repeat/Yearly/On.js

```diff
@@ -1,64 +1,78 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import moment from 'moment';
+
 import numericalFieldHandler from '../../../utils/numericalFieldHandler';
 
 const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
```

Comment:
BTW The same array is in `src/lib/utils/computeRRule/computeYearlyOn.js`.
What about extracting this and other hardcoded data you know about, into `lib/constants` ?

Reply:
I thought this day will finally arrive to have a purpose moving it to constants. This day arrived already. Moved to constants. ;)